src/v/archival/archival_policy.cc

```diff
@@ -36,15 +39,17 @@ archival_policy::archival_policy(
   , _ntp_probe(ntp_probe) {}
 
 archival_policy::lookup_result archival_policy::find_segment(
-  model::offset last_offset, storage::log_manager& lm) {
+  model::offset last_offset,
+  model::offset high_watermark,
+  storage::log_manager& lm) {
     vlog(
-      archival_log.trace,
+      archival_log.debug,
```

Comment:
I see a lot of `log` entries changes. Is this intended ? If so let's make it separate commit

Reply:
done